components/ai_chat/resources/page/components/full_page/index.tsx

```diff
@@ -4,129 +4,158 @@
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react'
-import Button from '@brave/leo/react/button'
+import styles from './style.module.scss'
+import classnames from '$web-common/classnames'
 import Icon from '@brave/leo/react/icon'
-import useMediaQuery from '$web-common/useMediaQuery'
+import ButtonMenu from '@brave/leo/react/buttonMenu'
 import { useAIChat } from '../../state/ai_chat_context'
-import ConversationsList from '../conversations_list'
-import { NavigationHeader } from '../header'
-import Main from '../main'
-import styles from './style.module.scss'
-import { useActiveChat } from '../../state/active_chat_context'
-
-export default function FullScreen() {
```

Comment:
I'm going to need to build this, but if you've successfully got rid of this I'll be happy as!